pages/artist/_itemId/index.vue

```diff
@@ -76,7 +77,9 @@
                 <v-container>
                   <v-row>
                     <v-col>
-                      <v-img cover :src="getImageUrl(item.Id, 'Backdrop')" />
+                      <v-img v-if="item.Id" cover>
+                        <blurhash-image :item="item" :type="'Backdrop'" />
+                      </v-img>
```

Comment:
Why is `<blurhash-image />` inside a `<v-img>` here?

Reply:
Same as above